refinery/core/api.py

```diff
@@ -498,6 +498,13 @@ class Meta:
         }
 
     def dehydrate(self, bundle):
+        version_details = bundle.obj.get_version_details()
+
+        if version_details is None:
```

Comment:
Is this actually necessary if you already have added checks to `obj_get()` and `obj_get_list()`?